tracer/resources/args_parser.py

```diff
@@ -16,7 +16,7 @@
 # 02110-1301, USA.
 #
 
-import argparse
+import argparse, argcomplete
```

Comment:
Isn't Tracer in AUR also? I don't see the benefit of making it optional if the user doesn't have to enable an extra repo for it.